ipa-core/src/ff/prime_field.rs

```diff
@@ -231,6 +231,12 @@ macro_rules! field_impl {
             }
         }
 
+        impl Default for $field {
+            fn default() -> Self {
+                Self::ZERO
+            }
+        }
```

Comment:
We might benefit from a 

```rust
for F in [Fp25519, Fp31, ...] {
  #[test] fn default_is_zero() {
    assert_eq!(F::ZERO, F::default());
  }
}
```

Noting of course that you can't iterate over types like that to produce test functions...